src/server.js

```diff
@@ -25,6 +25,7 @@ Server.prototype.listen = function(port, host) {
     var client = new Client(true);
     client._end = client.end;
     client.end = function end(endReason) {
+      endReason='{"text":"'+endReason+'"}';
```

Comment:
Alright I just checked again, I get an error there https://github.com/PrismarineJS/node-minecraft-protocol/pull/212/files#diff-c1129c8b045390789fa8ff62f2c6b4a9R442 if I run this code in 1.8 : the 1.8 server really sends "Failed to verify username!"